policies/rules/security/entry.go

```diff
@@ -0,0 +1,750 @@
+				return nil, fmt.Errorf("profiles is nil")
+			}
+			obj := obj.Profiles
+			for _, chk := range []string{"profiles.", "Profiles."} {
+				if strings.HasPrefix(v, chk) {
+					v = v[len(chk):]
+					break
+				}
+			}
+
+			if v == "url_filtering_profiles" || v == "UrlFilteringProfiles" {
+				return obj.UrlFilteringProfiles, nil
+			}
+			if v == "url_filtering_profiles|LENGTH" || v == "UrlFilteringProfiles|LENGTH" {
+				return len(obj.UrlFilteringProfiles), nil
+			}
+			if v == "data_filtering_profiles" || v == "DataFilteringProfiles" {
+				return obj.DataFilteringProfiles, nil
+			}
+			if v == "data_filtering_profiles|LENGTH" || v == "DataFilteringProfiles|LENGTH" {
+				return len(obj.DataFilteringProfiles), nil
+			}
+			if v == "file_blocking_profiles" || v == "FileBlockingProfiles" {
+				return obj.FileBlockingProfiles, nil
+			}
+			if v == "file_blocking_profiles|LENGTH" || v == "FileBlockingProfiles|LENGTH" {
+				return len(obj.FileBlockingProfiles), nil
+			}
+			if v == "wildfire_analysis_profiles" || v == "WildfireAnalysisProfiles" {
+				return obj.WildfireAnalysisProfiles, nil
+			}
+			if v == "wildfire_analysis_profiles|LENGTH" || v == "WildfireAnalysisProfiles|LENGTH" {
+				return len(obj.WildfireAnalysisProfiles), nil
+			}
+			if v == "anti_virus_profiles" || v == "AntiVirusProfiles" {
+				return obj.AntiVirusProfiles, nil
+			}
+			if v == "anti_virus_profiles|LENGTH" || v == "AntiVirusProfiles|LENGTH" {
+				return len(obj.AntiVirusProfiles), nil
+			}
+			if v == "anti_spyware_profiles" || v == "AntiSpywareProfiles" {
+				return obj.AntiSpywareProfiles, nil
+			}
+			if v == "anti_spyware_profiles|LENGTH" || v == "AntiSpywareProfiles|LENGTH" {
+				return len(obj.AntiSpywareProfiles), nil
+			}
+			if v == "vulnerability_profiles" || v == "VulnerabilityProfiles" {
+				return obj.VulnerabilityProfiles, nil
+			}
+			if v == "vulnerability_profiles|LENGTH" || v == "VulnerabilityProfiles|LENGTH" {
+				return len(obj.VulnerabilityProfiles), nil
+			}
+		}
+	}
+	if v == "qos" || v == "Qos" {
+		return obj.Qos != nil, nil
+	}
+	if strings.HasPrefix(v, "qos.") || strings.HasPrefix(v, "Qos.") {
+		if obj.Qos == nil {
+			return nil, fmt.Errorf("qos is nil")
+		}
+		obj := obj.Qos
+		for _, chk := range []string{"qos.", "Qos."} {
+			if strings.HasPrefix(v, chk) {
+				v = v[len(chk):]
+				break
+			}
+		}
+
+		if v == "ip_dscp" || v == "IpDscp" {
+			return obj.IpDscp, nil
+		}
+		if v == "ip_precedence" || v == "IpPrecedence" {
+			return obj.IpPrecedence, nil
+		}
+		if v == "follow_client_to_server_flow" || v == "FollowClientToServerFlow" {
+			return obj.FollowClientToServerFlow != nil, nil
+		}
+	}
+	if v == "negate_target" || v == "NegateTarget" {
+		return e.NegateTarget, nil
+	}
+	if v == "disable_inspect" || v == "DisableInspect" {
+		return e.DisableInspect, nil
+	}
+
+	return nil, fmt.Errorf("unknown field: %s", value)
+}
+
+func Versioning(vn version.Number) (Specifier, Normalizer, error) {
```

Comment:
Code is commented here.

Reply:
Yes, this is because security rules should actually have a second versioned object for PAN-OS 10.2.0, but actually adding that support in to this PR felt like it would make the PR too unwieldy.  So this comment exists as a reminder that a second object for PAN-OS > 10.2.0 should exist, it's just not in this PR.  This is the `DisableInspect` param on line 59.